src/LayerSwitcher/LayerSwitcher.less

```diff
@@ -0,0 +1,29 @@
+.react-geo-layer-switcher {
+  .clip {
+    width: 80px;
+    height: 80px;
+    background-color: white;
+    border: 2px solid black;
+    border-radius: 4px;
+    overflow: hidden;
+    cursor: pointer;
+  }
+
+  #layer-switcher-map {
+    width: 256px;
+    height: 256px;
+    position: relative;
+    left: -68px;
+    top: -68px;
+    pointer-events: none;
+  }
+
+  .layer-title {
+    position: absolute;
+    width: 100%;
+    bottom: 0;
+    padding: 3px;
+    color: white;
+    background-image: linear-gradient(transparent,rgba(0,0,0,0.6));
```

Comment:
How does it look like if the title is too long? Will it be clipped? Can we use e.g. text-shadow to have the title be readable regardless of the actual layer?

Reply:
text-overflow is not yet tested. I will add it.

The background-image gradient is there to ensure the readability of the the title.